packages/style-engine/class-wp-style-engine.php

```diff
@@ -321,6 +471,11 @@ public function generate( $block_styles, $options ) {
  * Returns an CSS ruleset.
  * Styles are bundled based on the instructions in BLOCK_STYLE_DEFINITIONS_METADATA.
  *
+ * Example usage:
+ *
+ * $styles = gutenberg_style_engine_generate( array( 'color' => array( 'text' => '#cccccc' ) ) );
```

Comment:
Tiniest of tiny nits: should this be `wp_style_engine_generate` in the example?

Reply:
I'm pretty sure my [nit above](https://github.com/WordPress/gutenberg/pull/40531#discussion_r891915411) is tinier 🥇

Reply:
True, you win this round, Aaron! 😆

Reply:
10 points to Gryffindor!